tests/linux/2024.vncjew/__min__c.simple

```diff
@@ -1,13 +1,71 @@
-# linux/2024.vncjew/__min__c: critical
-anti-static/elf/entropy: high
```

Comment:
For UPX (or any other form of program obfuscation), what do you think about including rule results for both the obfuscated and unobfuscated forms? I feel like the obfuscation itself is a good clue we shouldn't discard.

Reply:
That's interesting. Would we have to scan the file twice then?

Reply:
Nevermind -- I just added `-k` to preserve the original file so that both it and the decompressed file are scanned: [`f849ff6` (#731)](https://github.com/chainguard-dev/malcontent/pull/731/commits/f849ff659f7acd25b7a2f71e2b643ea13b027088).